docs/src/main/resources/content/replication-configuring.adoc

```diff
@@ -0,0 +1,107 @@
+:title: Configuring Replication
+:type: configuration
+:status: published
+:parent: Replication
+:summary: Instructions for configuring replication.
+:order: 15
+
+== {title}
+
+Replication must be configured before running.
+The following sections describe the console commands that are provided for configuration.
+
+[TIP]
+====
+Details about these commands can also be viewed from the ${command-console} using the `--help` option.
+====
+
+=== Adding a Replication Configuration
+
+The preferred method of configuring replication is through the replication UI found at https://<host>:<port>/admin/replication
+
+Configuration can still be created and managed via the commandline as detailed below.
```

Comment:
@ricklarsen, we might need to check with @clockard I think some of these commands were taken out at a specific verison.

Reply:
yeah. It looks like I targeted the wrong branch